examples/protocol_specification_ex/output/two_party_negotiation/__init__.py

```diff
@@ -1 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
@marcofavorito this file is the output from the generator. It currently does not create a license header. We could do that but then the Copyright must be based on the author specified.